packages/client/src/client.rs

```diff
@@ -83,22 +83,6 @@ impl PolywrapClient {
 }
 
 impl Invoker for PolywrapClient {
-    fn invoke_wrapper_raw(
```

Comment:
I think we still want this on the PolywrapClient and Client trait (just not as part of invoker trait)